pkg/registry/apps/rest/storage_apps.go

```diff
@@ -86,9 +86,10 @@ func (p RESTStorageProvider) v1beta2Storage(apiResourceConfigSource serverstorag
 		storage["deployments/scale"] = deploymentStorage.Scale
 	}
 	if apiResourceConfigSource.ResourceEnabled(version.WithResource("statefulsets")) {
-		statefulsetStorage, statefulsetStatusStorage := statefulsetstore.NewREST(restOptionsGetter)
-		storage["statefulsets"] = statefulsetStorage
-		storage["statefulsets/status"] = statefulsetStatusStorage
+		statefulSetStorage := statefulsetstore.NewStorage(restOptionsGetter)
+		storage["statefulsets"] = statefulSetStorage.StatefulSet
+		storage["statefulsets/status"] = statefulSetStorage.Status
+		storage["statefulsets/scale"] = statefulSetStorage.Scale
```

Comment:
@liggitt we want to advance v1beta2 to v1. So we are putting our effort there. There is no reason to not add a scale subresource to v1beta1 (it would be backward compatible). Do you have a strong reason or concern that would motivate adding a scale subresource to v1beta1?